releasenotes/notes/1.2/fix-vf2-aer-a7306ce07ea81700.yaml

```diff
@@ -1,4 +1,4 @@
 fixes:
   - |
-    The :class:`.VF2Layout` pass would raise an exception when provided with a :class:`.Target` instance without connectivity constraints.
-    This would be the case with targets from Aer 0.13. The issue is now fixed.
+    Fixed the :class:`.VF2Layout` for :class:`.Target`\s without connectivity constraints 
```

Comment:
I think this was already released as part of 1.1.1. We probably should go through the release notes and check for duplicates on the `stable/1.1` branch to make sure we're not duplicating anything for 1.2. If we are we can just delete the duplicates.

I know there is a release note documenting the new classical variables feature which was from 1.1. It'll show up here because it merged to main after we tagged 1.1.0rc1 so reno doesn't see it for the 1.1.0 release.

Reply:
Very good catch, it's indeed been in 1.1.1 already. There's also some other ones, should they be deleted or rather just moved to `releasenotes/notes/1.1`?

Reply:
Just delete them, reno works with the git history not the directory. If the file shows it was added to the branch's git log after the last tag on the branch (which would be 1.1.0rc1). It will still think that after a move.